### PR TITLE
Percy fix

### PR DESCRIPTION
**Pull request by ethangj**

### CODE REVIEW

File: _general/integrations/percy.md [hunk 1]
```diff
@@ -19,6 +19,9 @@ redirect_from:
 * include a table of contents
 {:toc}
 
+<div class="warning-block">
+Because we don't have a pull request identifier environment variable, Percy integration is limited in it's ability to generate screenshot comparisons over time.</div>
```
Comment (joesiewert) [inline _general/integrations/percy.md]: `the Percy integration`

Comment (joesiewert) [inline _general/integrations/percy.md]: `it's` to `its` in this case I think

Comment (joesiewert) [reply _general/integrations/percy.md]: IIRC `</div>` needs to be on a separate line to render correctly

**Comment by ethangj** [comment]
@joesiewert Updated

**Pull request merged**